srcpkgs/lego/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'lego'
+pkgname=lego
+version=2.4.0
+revision=1
+archs="i686 x86_64 armv7 aarch64"
```

Comment:
uh, shouldn't it also be in musl ?

Reply:
if there is no specific reason, just drop this line to enable all architectures